### PR TITLE
feat: add script for generating java package

### PR DESCRIPTION
**Pull request by AntonBulavko**

### CODE REVIEW

File: CreateJavaPackage.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		curl -k --http1.1 --connect-timeout 30 --retry 300 --retry-delay 5 --retry-connrefused $specurl > ./spec.json	;;
+    *)
+		echo Copying spec file
+		cp $specurl ./spec.json ;;
+esac
+
+cp /openapitools.json ./openapitools.json
+
+echo Generating API
+npx openapi-generator generate -i ./spec.json -g java -o java_service --git-user-id $repoOwner --git-repo-id $repoId --global-property models,modelTests=false,modelDocs=false -p basePackage=mqube.caseService -p modelPackage=mqube.caseService.models -p dateLibrary=java8
```
Comment (craggs) [inline CreateJavaPackage.sh:27]: shouldnt this be `npx openapi-generator-cli generate`  (note the extra `-cli`)
 - thats how the latest version of open api gen is used :) 
 https://github.com/spring-financial-group/jx3-openapi-generation/blob/master/CreateAngularPackageV3.sh#L24

**Review by craggs — CHANGES_REQUESTED** [review]

File: CreateJavaPackage.sh (hunk 1 above)
Comment (AlexKondrat) [reply CreateJavaPackage.sh:27]: Thanks Paul!

**Review by AlexKondrat — APPROVED** [review]

File: CreateJavaPackage.sh (hunk 1 above)
Comment (AlexKondrat) [reply CreateJavaPackage.sh]: Fixed :)

**Pull request merged**